2.3.1-ubi7/Dockerfile

```diff
@@ -0,0 +1,122 @@
+COPY bintray-apache-couchdb-rpm.repo /etc/yum.repos.d/bintray-apache-couchdb-rpm.repo
+
+ENV COUCHDB_VERSION 2.3.1
+
+# Add CouchDB user account to make sure the IDs are assigned consistently
+# CouchDB user added to root group for OpenShift support
+RUN set -ex; \
+    useradd -u 5984 -d /opt/couchdb -g root couchdb; \
+# be sure GPG and apt-transport-https are available and functional
+    yum update --disableplugin=subscription-manager -y && rm -rf /var/cache/yum; \
+    yum install -y \
+            ca-certificates \
+            gnupg \
+            pygpgme; \
+# install wget
+    yum update --disableplugin=subscription-manager -y && rm -rf /var/cache/yum; \
+    yum history new; \
+    yum install -y wget; \
+# Enable EPEL repositories (needed for python-progressbar dependency)
+    wget http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm; \
+# Remove wget
+    yum -y history undo 1; \
+    yum install -y epel-release-latest-7.noarch.rpm; \
+    rm epel-release-latest-7.noarch.rpm; \
+# Install runit
+     yum update --disableplugin=subscription-manager -y && rm -rf /var/cache/yum; \
+     yum install --enablerepo=imeyer_runit -y runit; \
+# Clean up
+    yum clean all; \
+    rm -rf /var/cache/yum
+
+# https://docs.couchdb.org/en/stable/install/unix.html
+# ENV GPG_COUCH_KEY \
+# # gpg: key D401AB61: public key "Bintray (by JFrog) <bintray@bintray.com> imported
+#        8756C4F765C9AC3CB6B85D62379CE192D401AB61
+# RUN set -xe; \
+#         export GNUPGHOME="$(mktemp -d)"; \
+#         echo "disable-ipv6" >> ${GNUPGHOME}/dirmngr.conf; \
+#         for server in $(shuf -e pgpkeys.mit.edu \
+#             ha.pool.sks-keyservers.net \
+#             hkp://p80.pool.sks-keyservers.net:80 \
+#             pgp.mit.edu) ; do \
+#                 gpg --batch --keyserver $server --recv-keys $GPG_COUCH_KEY && break || : ; \
+#         done; \
+#         gpg --batch --export $GPG_COUCH_KEY > /etc/apt/trusted.gpg.d/couchdb.gpg; \
+#         command -v gpgconf && gpgconf --kill all || :; \
+#         rm -rf "$GNUPGHOME"; \
+#         apt-key list
+
+# Install CouchDB
+RUN set -xe; \
+    yum update --disableplugin=subscription-manager -y && rm -rf /var/cache/yum; \
+    yum install --enablerepo=bintray-apache-couchdb-rpm -y couchdb; \
+    yum clean all; \
+    rm -rf /var/cache/yum; \
+# remove defaults that force writing logs to file
+    rm /opt/couchdb/etc/default.d/10-filelog.ini; \
+# Check we own everything in /opt/couchdb. Matches the command in dockerfile_entrypoint.sh
+    find /opt/couchdb \! \( -user couchdb -group 0 \) -exec chown -f couchdb:0 '{}' +; \
+# Setup directories and permissions for config. Technically these could be 555 and 444 respectively
+# but we keep them as 755 and 644 for consistency with CouchDB defaults and the dockerfile_entrypoint.
+    find /opt/couchdb/etc -type d ! -perm 0755 -exec chmod -f 0775 '{}' +; \
+    find /opt/couchdb/etc -type f ! -perm 0644 -exec chmod -f 0664 '{}' +; \
```

Comment:
Should the check here be looking for 0775 / 0664 instead of 0755 / 0644? Looks like it might have been an oversight in the switch to the root group.